### PR TITLE
feature/155_ci_build_3_9

### PR DESCRIPTION
**Pull request by damian-romero**
See #155

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,7 +61,7 @@ These instructions will have the code running on your local or virtual machine.
 
 You will need the following to have a working copy of this software. See
 [installation](#installation-instructions) steps:
-1. Python (3.7.5)
+1. Python >= v3.7.9
```
Comment (astrochun) [inline README.md:64]:
```suggestion
1. Python (>=v3.7.9)
```

File: README.md [hunk 2]
```diff
@@ -61,7 +61,7 @@ These instructions will have the code running on your local or virtual machine.
...
 2. [`figshare`](https://github.com/ualibraries/figshare) - UA Libraries' forked copy of [cognoma's figshare](https://github.com/cognoma/figshare)
 3. [`pandas`](https://pandas.pydata.org/) ([1.0.2](https://pandas.pydata.org/pandas-docs/version/1.0.2/))
```
Comment (astrochun) [inline README.md:66]:
```suggestion
3. [`pandas`](https://pandas.pydata.org/) ([1.2.1](https://pandas.pydata.org/pandas-docs/version/1.2.1/))
```

File: README.md [hunk 3]
```diff
@@ -74,14 +74,14 @@ You will need the following to have a working copy of this software. See
 
 #### Python and setting up a `conda` environment
 
-First, install a working version of Python (v3.7.5).  We recommend using the
+First, install a working version of Python >= v3.7.9.  We recommend using the
```
Comment (astrochun) [inline README.md:77]:
```suggestion
First, install a working version of Python (>=3.7.9).  We recommend using the
```

**Review by astrochun — CHANGES_REQUESTED** [review]
Changes look good. Just a few suggestions to README.md.  Includes update to `pandas` version.

Still need to update `numpy` version

File: README.md [hunk 4]
```diff
@@ -61,11 +61,11 @@ These instructions will have the code running on your local or virtual machine.
...
 You will need the following to have a working copy of this software. See
 [installation](#installation-instructions) steps:
-1. Python (3.7.5)
+1. Python (>=v3.7.9)
 2. [`figshare`](https://github.com/ualibraries/figshare) - UA Libraries' forked copy of [cognoma's figshare](https://github.com/cognoma/figshare)
-3. [`pandas`](https://pandas.pydata.org/) ([1.0.2](https://pandas.pydata.org/pandas-docs/version/1.0.2/))
+3. [`pandas`](https://pandas.pydata.org/) ([1.2.1](https://pandas.pydata.org/pandas-docs/version/1.2.1/))
 4. [`requests`](https://requests.readthedocs.io/en/master/) ([2.22.0](https://requests.readthedocs.io/en/master/2.22.0))
-5. [`numpy`](https://numpy.org/) ([1.17.4](https://numpy.org/devdocs/release/1.17.4-notes.html/1.17.4))
+5. [`numpy`](https://numpy.org/) ([1.20.0](https://numpy.org/devdocs/release/1.20.0-notes.html)).17.4))
```
Comment (damian-romero) [inline README.md:68]: Fix typo
```suggestion
5. [`numpy`](https://numpy.org/) ([1.20.0](https://numpy.org/devdocs/release/1.20.0-notes.html))
```

**Review by astrochun — APPROVED** [review]
Changes look good.

**Pull request merged**